master/usage/assigning-ip-pools-per-node.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Assigning IP pools per-Node
+---
+
+## About IP pool Node selection
+
+Starting from {{site.prodname}} {{site.min-versions.ip-pool-node-select}}, IP pools can be configured 
+to restrict IP address assignment of workloads to specific Nodes. This feature is useful for
+network/system administrators that are interested in assigning IPs to pods based on rack affinity.
+This guide walkthroughs an example of how to setup and manage this feature.
+
+## Prerequisites
+
+**{{site.prodname}} IPAM**
+
+This guide only applies if you are using {{site.prodname}} IPAM.
+
+**Labeled Nodes**
+
+In order to assign IP pools to specific Nodes, these Nodes must be labeled. See the documentation for [calicoctl label]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/label) and [kubectl label](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes/#add-a-label-to-a-node) for more information on how to do this. We recommend using `kubectl label`.
+
+### Example: Kubernetes
+
+In this example, we created a cluster with 8 nodes across 4 racks (2 nodes/rack). Consider the following:
+
+```
+       -------------------------------------------------------
+       |                router                               |
+       -------------------------------------------------------
+       /                 |                 |                 \
+---------------   ---------------   ---------------   ---------------   
+| rack-0      |   | rack-1      |   | rack-2      |   | rack-3      |   
+---------------   ---------------   ---------------   ---------------   
+| kube-node-0 |   | kube-node-2 |   | kube-node-4 |   | kube-node-6 |   
+- - - - - - - -   - - - - - - - -   - - - - - - - -   - - - - - - - -   
+| kube-node-1 |   | kube-node-3 |   | kube-node-5 |   | kube-node-7 |   
+- - - - - - - -   - - - - - - - -   - - - - - - - -   - - - - - - - -   
+```
+
+Using the pods IP range `192.168.0.0/16`, we will want to optimize our IP allocation strategy to assign IPs from 4 different pools for each rack.
+For this example, we target the following setup: `192.168.0.0/18`, `192.168.64.0/18`, `192.168.128.0/18`, and `192.168.196.0/18` blocks
+for `rack-0`, `rack-1`, `rack-2`, and `rack-3` respectively. Let's get started.
+
+
+By installing {{ site.prodname }} without setting the
```

Comment:
Well, without the /16 pool being disabled in some way, it won't be possible to create the smaller pools. Should we have the user disable the pool instead of outright delete it?